securedrop/tests/test_journalist.py

```diff
@@ -113,19 +106,16 @@ def tearDown(self):
 
     @patch('db.Journalist._scrypt_hash')
     @patch('db.Journalist.valid_password', return_value=True)
-    def test_login_with_valid_length_password_calls_scrypt(
-            self, mock_scrypt_hash, mock_valid_password):
-        self.login(self.username, self.password)
+    def test_login_calls_scrypt(self, mock_scrypt_hash, mock_valid_password):
```

Comment:
Maybe `test_valid_login_calls_scrypt`? There are multiple unit tests for `login` testing `scrypt` calls, good to be precise here